tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/srcdoc/consecutive-srcdoc.html.ini

```diff
@@ -1,6 +1,7 @@
 [consecutive-srcdoc.html]
+  expected: TIMEOUT
```

Comment:
I think it was intermittent TIMEOUT?

```
1 unexpected results that are NOT known-intermittents:
  ▶ OK [expected TIMEOUT] b'/html/browsers/browsing-the-web/history-traversal/srcdoc/consecutive-srcdoc.html'

  ▶ Unexpected subtest result in /html/browsers/browsing-the-web/history-traversal/srcdoc/consecutive-srcdoc.html:
  │ FAIL [expected TIMEOUT] b'changing srcdoc does a replace navigation since the URL is still about:srcdoc'
  └   → promise_test: Unhandled rejection with value: object "TypeError: iframe is null"

  ▶ Unexpected subtest result in /html/browsers/browsing-the-web/history-traversal/srcdoc/consecutive-srcdoc.html:
  │ FAIL [expected NOTRUN] b'changing srcdoc to about:srcdoc#yo then another srcdoc does two push navigations and we can navigate back'
  └   → promise_test: Unhandled rejection with value: object "TypeError: iframe is null"
```

Reply:
Filed https://github.com/servo/servo/issues/29084 and we'll need to update the test status in this PR 🙏 please feel free to help me close the issue if I'm wrong 🙇

Reply:
Considering that this PR only contains metadata changes, no actual test changes, yes, this is an intermittent result :)